app/src/main/java/com/amaze/filemanager/ui/Extensions.kt

```diff
@@ -53,6 +55,23 @@ fun Context.startActivityCatchingSecurityException(intent: Intent) {
     }
 }
 
+/**
+ * Refactor the alias, based on if Amaze Utils is installed, then we disable the alias intent-filters
```

Comment:
What do you mean by refractor? Rename maybe? Refactor makes sense only for code.